Update references in README (#46)
IBM/sample-app-mod: README.md

- `liberty-java8`
  This branch shows what the application looks like after it has been modernized to Liberty. Comparing this branch to main, you will notice the following changes:
  - Code changes in some source files (to remove use of WAS APIs)

>   - Addition of the Liberty config file: `src/main/liberty/config/server.xml`. This file is produced by IBM Application Modernization Accelerator and is available in the [migration bundle](#migration-bundle)

Reviewer: ```suggestion
  - Addition of the Liberty config file: `src/main/liberty/config/server.xml`. This file is produced by IBM Application Modernization Accelerator and is available in the [migration plan](#migration-plan)
```

There are no special instructions for deploying the application to IBM WebSphere Application Server. There is no configuration required on the application server in order for the application to deploy and function.

It can be deployed using the UI console or using `wsadmin`.

> Refer to the [documentation](https://www.ibm.com/docs/en/was-nd/9.0.5?topic=applications-how-do-i-deploy) for more details on deploying the application to WebSphere Application Server.

Reviewer: ```suggestion
See the [documentation](https://www.ibm.com/docs/en/was-nd/9.0.5?topic=applications-how-do-i-deploy) for more details on deploying the application to WebSphere Application Server.
```

## Deploying the Application to Liberty
To deploy the application on Liberty you can do one of the following:
- Install the Liberty Tools IDE plugin (available for VS Code, Eclipse IDE and IntelliJ IDEA)
- Add Liberty Maven or Gradle Plugin configuration to your build file. Note, for convenience, configuration for Liberty Maven Plugin is already added to the `pom.xml` in the `liberty-` branches. Liberty can be launched in dev mode with the following command:
```
mvn liberty:dev
```

> - Launch Liberty in dev mode directly from command line:

Reviewer: ```suggestion
- Start Liberty in dev mode directly from command line:
```

```
mvn io.openliberty.tools:liberty-maven-plugin:3.10.2:dev
```
- Build and drop the WAR file into Liberty installation.

> For more on Liberty Tools, refer to [Develop with Liberty Tools](https://openliberty.io/docs/latest/develop-liberty-tools.html)

Reviewer: ```suggestion
For more on Liberty Tools, see [Develop with Liberty Tools](https://openliberty.io/docs/latest/develop-liberty-tools.html)
```

## Building and Running the Application a Liberty Container

> A Containerfile exists in the `liberty-` branches. The Containerfile is produced by IBM Application Modernization Accelerator and is available in the [migration bundle](#migration-bundle). It can be used to build an image and run the application in a container. You can build the image as follows:

Reviewer: ```suggestion
A Containerfile exists in the `liberty-` branches. The Containerfile is produced by IBM Application Modernization Accelerator and is available in the [migration plan](#migration-plan). It can be used to build an image and run the application in a container. You can build the image as follows:
```

```

> ## Migration Bundle

Reviewer: ```suggestion
## Migration Plan
```

> The `migration_bundle` directory contains a migration bundle for ModResorts created by [IBM Application Modernization Accelerator](https://www.ibm.com/products/jsphere/tools) (AMA). It is used by IBM AMA Dev Tools (available for VS Code, Eclipse IDE, and IntelliJ IDEA) or IBM Bob to accelerate modernization of legacy enterprise Java applications to run on Liberty.  [Try](https://www.ibm.com/account/reg/us-en/signup?formid=urx-53705) AMA.

Reviewer: ```suggestion
The `migration_bundle` directory contains a migration plan for ModResorts created by [IBM Application Modernization Accelerator](https://www.ibm.com/products/jsphere/tools) (AMA). It is used by IBM AMA Dev Tools (available for VS Code, Eclipse IDE, and IntelliJ IDEA) or IBM Bob to accelerate modernization of legacy enterprise Java applications to run on Liberty.  [Try](https://www.ibm.com/account/reg/us-en/signup?formid=urx-53705) AMA.
```

Reviewer 2: OK, lets get this one merged, and i'l follow up with a PR to change the directory and file and remaining reference.

Reviewer: There seems to be a change blocking it still. Can't see which one...
